### ISSUE TITLE
Misplaced settings button when DPI is changed using adb

### ISSUE THREAD

**Issue by FreakyEnderman** [wasdennnoch/AndroidN-ify#1155]
https://postimg.org/image/lwvzj7yg1/

This is how it looks. (Also an example of GB bug)

**Comment by FreakyEnderman** [comment]
Note that it might not because of custom DPI. I haven't tested enough because I am at work at the moment. Sorry for that.

**Comment by wasdennnoch** [comment]
Have you rebooted your phone after that?

**Comment by FreakyEnderman** [comment]
Yes I have. I haven't tried other DPIs though maybe there is something going on about the DPI that I am using.

**Comment by FreakyEnderman** [comment]
Nope that ain't it. Settings icon is still misplaced. Worked well before in Lollipop. I clean flashed Marshmallow to my Xperia and icon is misplaced.

**Comment by FreakyEnderman** [comment]
Well ignorance is not going to let me provide a log. What log or logs do you need. I must search how to get them that's why I am asking. Forgive my ignorance.

**Comment by FreakyEnderman** [comment]
No error logs in xposed by the way.

**Comment by wasdennnoch** [comment]
Really, no errors in the Xposed log? The header layout looks completely screwed.  Please post it just to be sure.

**Comment by FreakyEnderman** [comment]
Only problem is the settings button I have removed the date and time using paint so that isn't someting to be worried about.

I can't attach the empty log because it is empty so uploading system wont let me. It just says that it's empty and that I can't attach it since it is empty.

**Comment by wasdennnoch** [comment]
Oh, I thought the panel was empty. If the Xposed log is empty you can try to post your logcat immediately after a reboot (or alternatively a SystemUI restart).

**Comment by FreakyEnderman** [comment]
![screenshot_20161105-235642](https://cloud.githubusercontent.com/assets/10789942/20033862/a8ca958a-a3b3-11e6-8160-398733833d26.png)

THis is how it looks without any edit. I will try to get a logcat and post it.

**Comment by FreakyEnderman** [comment]
[logcat.txt](https://github.com/wasdennnoch/AndroidN-ify/files/573537/logcat.txt)


As I understood from
"[ 11-06 00:03:45.688  9036: 9036 I/ActivityThread ]
Switching default density from 480 to 350" this,
device sets screen density to my custom one upon every boot which might cause module to think that I am actually at the original DPI.

**Comment by FreakyEnderman** [comment]
This DPI issue might be caused because of my ROM because CallUI buttons and picture of the caller is misplaced too. (It was the same in Lollipop but the settings icon was fine)

**Comment by wasdennnoch** [comment]
Which ROM do you have?

**Comment by FreakyEnderman** [comment]
Sony's stock marshmallow

**Comment by FreakyEnderman** [comment]
Also reducing DPI is not adding another icon to my launcher (Pixel Launcher) and it supposed to because it did in Lollipop. So i think it is more rom centered issue.

**Comment by FreakyEnderman** [comment]
Update: Default DPI has not solved this issue. The settings button is still misplaced. The issue should be at Android N-ify side.

**Comment by wasdennnoch** [comment]
I guess I'll need your SystemUI then.

**Comment by FreakyEnderman** [comment]
Will just pulling it from system folder be enough?

**Comment by wasdennnoch** [comment]
I'll take a look at the lyout first, so just pulling it should do it. I'll ask for a deodexed version if the layout won't be enough.

**Comment by FreakyEnderman** [comment]
ROM is already deodexed it looks like. Odex folder exist inside.

**Comment by FreakyEnderman** [comment]
[https://drive.google.com/open?id=0B69aqj_vEvq3elhRQ2pSWDhEbTA](This)
should get you to my SytemUI folder at Google Drive

**Comment by FreakyEnderman** [comment]
By the way will this deodexed ROM (there is an oat file in each system app and there is arm64 folder inside there is an odex file) be problematic with modules? If it is problematic I will deodex the whole thing. It is an easy process anyway.

**Comment by wasdennnoch** [comment]
odex is no issue, it only means you can't decompile the app when odexed, but it has no impact on Xposed.

**Comment by FreakyEnderman** [comment]
Glad to hear that. I have uploaded my SystemUI if you need a deodexed version I will be able to deodex using something from xda. (I don't know what yet)
